src/site/mdbook.rs

```diff
@@ -128,7 +142,9 @@ impl AxomdbookTheme {
     pub fn name(&self) -> &'static str {
         use AxomdbookTheme::*;
         match self {
-            Axo => "Axo Dark",
+            Default => "Default Dark",
+            DefaultLight => "Default Light",
```

Comment:
These should probably be "Oranda Dark" and "Oranda Light"

Reply:
i feel complicated about this- i sorta like default because it helps me know it's the default- can you say more about why you prefer oranda? (i can imagine way, but am curious to hear you articulate it)

Reply:
because this is showing up in the dropdown menu where you select themes, and it's not The Default For Mdbook Itself, so it's a little weird.

Reply:
(nb mdbook already has themes called "dark" and "light")

Reply:
ahhhh yeah, i was forgetting that we still show all the themes (ya?) if so then yeah we should update this